README.md

```diff
@@ -7,3 +7,126 @@ Stages:
 2. Instead of using an image that has curl, make it have a server and present a page that consumes the CR
 3. Instead of this simple page, use Portafly
 4. [Optional] Authorize the calls. We may use Keycloak. We might also explore using only OpenShift RBAC and users.
+
+
+## Setup
+
+### Overview
+To run this project you need acces to a kubernetes cluster.
+We are using an **OpenShift 4.5** cluster, but you'd be able to use another kubernetes cluster.
+
+You can use both [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) or [OpenShift cli](https://docs.openshift.com/container-platform/4.5/cli_reference/openshift_cli/getting-started-cli.html) tools, in this document we use the OpenShift cli, but you may use `kubectl` instead of `oc`.
+
+### Creating a namespace
+
+To avoid namespace pollution, create a [kubernetes namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) called *k8sinitiative* in your cluster: 
```

Comment:
I'd keep it as `namespace`, as it applies to both `kubernetes` and `openshift`. `Project` is an openshift concept only

Reply:
Yup, I meant the redaction... instead of "to avoid namespace pollution..." something like "To avoid using your default namespace or previous working namespace..."